### PR TITLE
MODIFY : Modifies README to add workaround for missing webrick dependency error during jekyll serve

### PR DESCRIPTION
**Pull request by akmalick**
…ency juring jekyll serve

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -53,6 +53,9 @@ jekyll serve
 bundle exec jekyll serve
 ```
 
+**NOTE:** If the above serve command throws error saying `require': cannot load such file -- webrick (LoadError)` try to run `bundle add webrick` (automatically adds webrick gem to the Gemfile) or manually add `gem "webrick"` line to the Gemfile and then run the serve command again.
```
Comment (vsoch) [inline README.md:56]: This is a great tidbit! Here is a tiny tweak/suggestion:

```suggestion
**NOTE:** If the above serve command throws an error saying `require': cannot load such file -- webrick (LoadError)` try to run `bundle add webrick` to automatically add the webrick gem to your Gemfile, or manually add `gem "webrick"` line to the Gemfile and then run the serve command again.
```

Comment (akmalick) [reply README.md]: Sounds good. Done

**Review by vsoch — APPROVED** [review]

**Comment by vsoch** [comment]
Thank you @akmalick !

**Comment by akmalick** [comment]
> Thank you @akmalick !

Glad to have done that. Thank you @vsoch for making our lives easier. Really appreciate the effort.

**Pull request merged**